lib/src/model/broadcast/broadcast_repository.dart

```diff
@@ -0,0 +1,89 @@
+import 'package:collection/collection.dart';
+import 'package:deep_pick/deep_pick.dart';
+import 'package:fast_immutable_collections/fast_immutable_collections.dart';
+import 'package:lichess_mobile/src/model/broadcast/broadcast.dart';
+import 'package:lichess_mobile/src/model/common/chess.dart';
+import 'package:lichess_mobile/src/model/common/http.dart';
+import 'package:lichess_mobile/src/utils/json.dart';
+
+class BroadcastRepository {
+  BroadcastRepository(this.client);
+
+  final LichessClient client;
+
+  Future<IList<Broadcast>> getBroadcasts() {
+    return client.readNdJsonList(
+      Uri(path: '/api/broadcast'),
+      headers: {'Accept': 'application/x-ndjson'},
+      mapper: _makeBroadcastFromJson,
+      compare: (Broadcast a, Broadcast b) => b.priority.compareTo(a.priority),
```

Comment:
I believe the sorting could be done server side.

Reply:
I asked to backend developers if they could make the sort on server side